src/poliastro/twobody/orbit.py

```diff
@@ -749,48 +749,90 @@ def circular(
 
         Parameters
         ----------
         attractor : Body
             Main attractor.
-        angular_velocity : ~astropy.units.Quantity
-            Rotational angular velocity of the attractor.
-        period : ~astropy.units.Quantity
-            Attractor's rotational period, ignored if angular_velocity is passed.
-        hill_radius : ~astropy.units.Quantity
-            Radius of Hill sphere of the attractor (optional). Hill sphere radius(in
-            contrast with Laplace's SOI) is used here to validate the stability of the
-            geostationary orbit, that is to make sure that the orbital radius required
-            for the geostationary orbit is not outside of the gravitational sphere of
-            influence of the attractor.
-            Hill SOI of parent(if exists) of the attractor is ignored if hill_radius is not provided.
+
         """
+        return cls.synchronous(attractor)
 
-        if angular_velocity is None and period is None:
-            raise ValueError(
-                "At least one among angular_velocity or period must be passed"
-            )
+    @classmethod
+    @u.quantity_input(
+        a=u.km,
+        ecc=u.one,
+        inc=u.deg,
+        argp=u.deg,
+        arglat=u.deg,
+        raan=u.deg,
+        period_mul=u.one,
+    )
+    def synchronous(
+        cls,
+        attractor,
+        a=None,
+        ecc=0 * u.one,
+        inc=0 * u.deg,
+        argp=0 * u.deg,
+        arglat=0 * u.deg,
+        raan=0 * u.deg,
+        epoch=J2000,
+        plane=Planes.EARTH_EQUATOR,
+        period_mul=1 * u.one,
+    ):
+        r""" Returns an orbit where the orbital period equals the rotation rate of the orbited body.
+        The synchronous altitude for any central body can directly be obtained from Kepler's Third Law by setting
+        the orbit period P\ :sub:`sync`, equal to the rotation period of the central body relative to the
+        fixed stars D\ :sup:`*`.
+        In order to obtain this, it's important to match orbital period with sidereal rotation period.
+        Thus:
+        .. math::
 
-        if angular_velocity is None:
-            angular_velocity = 2 * np.pi / period
+                P_{s y n c}=D^{*} \\
 
-        # Find out geostationary radius using r = cube_root(GM/(angular
-        # velocity)^2)
-        with u.set_enabled_equivalencies(u.dimensionless_angles()):
-            geo_radius = np.cbrt(attractor.k / np.square(angular_velocity.to(1 / u.s)))
+                a_{s y n c}=\left(\mu / 4 \pi^{2}\right)^{1 / 3}\left(D^{*}\right)^{2 / 3}\\
 
-        if hill_radius is not None and geo_radius > hill_radius:
-            raise ValueError(
-                "Geostationary orbit for the given parameters doesn't exist"
-            )
+                H_{s y n c}=a_{s y n c} - R_{p l a n e t}\\
+
+       Parameters
+        ----------
+        attractor : Body
+            Main attractor.
```

Comment:
I think there's a whitespace issue:

```suggestion
       Parameters
       ----------
       attractor : Body
           Main attractor.
```